### ISSUE TITLE
3.2.5 Chromium Framework compatibility

### ISSUE THREAD

**Issue by MOLOTOFBOMB** [TheCheatsrichter/Gw2_Launchbuddy#427]
18.04.2023 Update after this program not work.
Ty

**Comment by luzclasil** [comment]
Not working here as well.
I'll add more details: after the latest update (gw2 ver. 144476) LB is unable to detect a loginfile update, locking itself in a loop.
If user closes the message, it says update was manually cancelled or something alike.
Trying to recreate the loginfile manually also did not work for me...

**Comment by gummbum** [comment]
As a workaround you can:

1. Step through all the loginfile updates by clicking Cancel for each one. They will be skipped.
2. The main Launchbuddy GUI will appear. Click the login you want to use and start it.
3. The loginfile update will be attempted again. Click Hide.
4. Manually use the GW2 launcher to: a) Login; and b) Play.

Note that the loginfile never completes the update, but at least you can circumvent the hang and still play using the Launchbuddy and the current (stale) loginfile.

**Comment by wikiidoo** [comment]
> As a workaround you can:
> 
> 1. Step through all the loginfile updates by clicking Cancel for each one. They will be skipped.
> 2. The main Launchbuddy GUI will appear. Click the login you want to use and start it.
> 3. The loginfile update will be attempted again. Click Hide.
> 4. Manually use the GW2 launcher to: a) Login; and b) Play.
> 
> Note that the loginfile never completes the update, but at least you can circumvent the hang and still play using the Launchbuddy and the current (stale) loginfile.

sadly this doesn't work.  scrapped all the account info and tried to start from scratch but still hangs on one account login.

**Comment by rh4242** [comment]
For what it is worth I did manage to get the current Launch Buddy to work with the following steps.

While experimenting I found that launch buddy version 3.0.5 would allow me to update the login files to for the current patch. It often takes 20-30 seconds per account so it may be waiting for a timeout, but it did work in the end. I do not know if this is needed but figured I would mention it.

1. Start the CURRENT version of Launch Buddy.
2. Open the account settings tab.
3. Expand the "Arguments" section.
4. Add -usecoherent as a custom argument. Make sure that it shows as checked and active.
5. Return to the main account listing and start that GW2 account.

If this works for you then each account in Launch Buddy will need to have the -usecoherent argument added. This should locally disable the new chromium framework that is causing the issues and some users on the GW2 forums have reported that this has temporarily fixed issues where the login screen is not working after the last update.

I was able to start and manage my 3 accounts normally after this.

**Comment by Pekols** [comment]
> 

Works for me. Thx

**Comment by MegaFart** [comment]
Same concern after the update using the latest LB version. I tried re-installing the game client and deleting the existing log in credentials in LB. Now it's not letting me save log in credentials to utilize multiple game clients. None of the above work arounds suggested helped so far. Hopefully this gets resolved soon.

**Comment by billypumphrey** [comment]
I have the same issue.

**Comment by billypumphrey** [comment]
The last time I checked, I believe using a .bat file to open the game have been deprecated, not working.  Does ayone still use a .bat file as a backup?  Or any other backup plan?

**Comment by billypumphrey** [comment]
I was able to download version 2.4.1 and it is working.

**Comment by jwreplogle** [comment]
I did the below and works with current version.

Replacing <gw2build>######</gw2build> with <gw2build>144476</gw2build>in the %APPDATA%\Gw2 Launchbuddy\Accs.xml

AND

Start the CURRENT version of Launch Buddy.
Open the account settings tab.
Expand the "Arguments" section.
Add -usecoherent as a custom argument. Make sure that it shows as checked and active.
Return to the main account listing and start that GW2 account.

As @rh4242 mentioned

**Comment by gummbum** [comment]
@jwreplogle Didn't work for me.

I replaced <gw2build>\d\d\d\d\d\d</gw2build> with <gw2build>144476</gw2build>, and added the custom argument.

It still hangs on updating the loginfile. I let it sit for a couple minutes. I hit Cancel to bypass the loginfile update. Launching a client causes LB to crash, presumably because of a problem concerning the loginfile.

**Comment by gummbum** [comment]
@rh4242 
Your workaround succeeded.

- launch buddy version 3.0.5
- add -usecoherent as a custom arg

I just have to tell LB "No" to updating to 3.2.4. And manually click Login and Play in the client launcher. Minor annoyances compared to playing without LB.

Thanks!

**Comment by zanagraf** [comment]
> @rh4242 Your workaround succeeded.
> 
> * launch buddy version 3.0.5
> * add -usecoherent as a custom arg
> 
> I just have to tell LB "No" to updating to 3.2.4. And manually click Login and Play in the client launcher. Minor annoyances compared to playing without LB.
> 
> Thanks!

Didn't work for me ! ;( #UpdateNeed

**Comment by jwreplogle** [comment]
> @jwreplogle Didn't work for me.
> 
> I replaced `<gw2build>\d\d\d\d\d\d</gw2build>` with `<gw2build>144476</gw2build>`, and added the custom argument.
> 
> It still hangs on updating the loginfile. I let it sit for a couple minutes. I hit Cancel to bypass the loginfile update. Launching a client causes LB to crash, presumably because of a problem concerning the loginfile.

Just in case you still need help on this version, make sure you replace all instances of the build number for <gw2build>######</gw2build>  If you get message to update then that instance needs to be changed also.

**Comment by TheCheatsrichter** [comment]
Small stability patch 3.2.5 is now available to get the basic functions back up running. Keep me updated if it is resolved now

**Comment by ubereffect** [comment]
Basic functions work for me, as of 3.2.5. Which is to say it doesn't get stuck on updating login file, and it can be used to launch multiple GW2 instances. Just one small hiccup that I've noticed so far. While the login screens come up prefilled, I need to press the gw2 'log in' button to actually launch the game.

**Comment by Split-Pea-Soup** [comment]
3.2.5 - Unable to edit Login credentials. Selecting either option (Set Loginfile or Manually Update Loginfile) opens the GW2 login screen for a moment, and then closes it before login credential can be entered/edited.

**Comment by jwreplogle** [comment]
went back to 3.2.4, i rather edit xml file myself and have each account launch automatically.  3.2.5 wil not launch automatically.

**Comment by Faye-Liyah** [comment]
I have tried the workarounds, but it doesn't bring back the functionality, that LB is meant to provide.

I started launch buddy and had to cancel all log in file updates.
Then closed LB and opened accs.xml and updated the gwclient numbers.
Started LB and it did not try to update log in files again.
Added -usecoherent to all Accounts as argument and checked the box.

What it does: It opens a single instance of the GW Launcher, even if I select more than one account. It does not fill out email and password, I have to enter it every time, for every account. I then have to click log in and play. While one game instance of GW2 is running i can't use LB at all it does not respond to any clicks. Starting a second instance does not work for me.

So, right now, LB is just a long complicated way to start a single account.
What LB should do, launch multiple accounts and enter the credentials, does not work at all for me.
